packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts

```diff
@@ -99,3 +181,15 @@ export function getApiUrlByBlockchainType(chain: Blockchain): string {
         return CROSSMINT_PROD_URL;
     }
 }
+
+export function getWeb3AuthBlockchain(chain: Blockchain): TORUS_LEGACY_NETWORK_TYPE {
```

Comment:
one last change, this method is identical to `getApiUrlByBlockchainType`. I suggest a small refactor.
a private method isTestNet(chain) with this logic returning a bool and later
```
getWeb3AuthBlockchain(chain: Blockchain) {
    return isTestNet(chain) ? WEB3_AUTH_TESTNET : WEB3_AUTH_MAINNET
}
```
and:
```
getApiUrlByBlockchainType(chain: Blockchain):
    return isTestNet(chain) ? CROSSMINT_STG_URL : CROSSMINT_PROD_URL
```

Reply:
Done